src/GraphQL.Tests/Bugs/Bug1156.cs

```diff
@@ -23,8 +23,7 @@ public void duplicated_type_names_should_throw_error()
 }";
             var result = AssertQueryWithErrors(query, null, expectedErrorCount: 1, executed: false);
             result.Errors[0].Message.ShouldBe("Error executing document.");
-            result.Errors[0].InnerException.Message.ShouldBe(@"Unable to register GraphType 'GraphQL.Tests.Bugs.Type2' with the name 'MyType'.
-The name 'MyType' is already registered to 'GraphQL.Tests.Bugs.Type1'. Check your schema configuration.");
+            result.Errors[0].InnerException.Message.ShouldBe(@"Unable to register GraphType 'GraphQL.Tests.Bugs.Type2' with the name 'MyType'. The name 'MyType' is already registered to 'GraphQL.Tests.Bugs.Type1'. Check your schema configuration.");
```

Comment:
Why newline was removed?

Reply:
New lines wreak havoc with Linux vs Windows. If you use Environment.NewLine then it is different on each platform, and you have to write tests to match. If not, it is different based on which one you use to build (since GitHub decodes files based on the platform).

Reply:
Besides I don’t think new lines belong in an exception message. I remove them whenever possible.

Reply:
OK